### PR TITLE
Tweak default camera.txt settings

### PR DESCRIPTION
**Pull request by htruong**
According to testing results in #23

### CODE REVIEW

File: package/piwebcam/camera.txt [hunk 1]
```diff
@@ -3,5 +3,9 @@
 # Use v4l2-ctl -L or have a look at camera.txt.example
 # to view all possible settings
 
-auto_exposure_bias=3
+auto_exposure=0
```
Comment (dgsiegel) [inline package/piwebcam/camera.txt:6]: ` auto_exposure=0` seems to be the default. Should we drop it?

File: package/piwebcam/camera.txt [hunk 2]
```diff
@@ -3,5 +3,9 @@
...
+auto_exposure_bias=12
```
Comment (dgsiegel) [inline package/piwebcam/camera.txt:7]: I think we could use 13 or 14 here as well? What do you think?

Comment (htruong) [reply package/piwebcam/camera.txt:6]: This is "good enough" for me.

File: package/piwebcam/camera.txt (hunk 1 above)
Comment (htruong) [reply package/piwebcam/camera.txt]: Changed it with newest push!

**Pull request merged**